articles/vmware/vmw-how-access-centos-update-servers.md

```diff
@@ -35,49 +35,41 @@ Our Elevated OFFICIAL security domain doesn't natively connect to the internet,
 
 ### Option 1. UKCloud-managed repository servers
 
-We provide repositories for CentOS 6 on our Elevated OFFICIAL security domain. To access them:
+We provide repositories for CentOS 6 and 7 on our Elevated OFFICIAL security domain. To access them:
 
 Create a file called `il3-repos.repo` in `/etc/yum.repos.d`, and populate it with the following:
 
-    ```
-    [base]
+```
+[base]
+name=UKCloud CentOS Repository - Base
+baseurl=http://<elevated-public-ip-address>/centos/<x>.<y>/os/x86_64
+gpgcheck=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-<x>
 
-    name=UKCloud CentOS Repository - Base
+[updates]
+name=UKCloud CentOS Repository - Updates
+baseurl=http://<elevated-public-ip-address>/centos/centos-updates
+gpgcheck=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-<x>
 
-    baseurl=http://<elevated-public-ip-address>/centos/6.5/os/x86_64
+[extras]
+name=UKCloud CentOS Repository - Extras
+baseurl=http://<elevated-public-ip-address>/centos/centos-extras
+gpgcheck=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-<x>
+```
 
-    gpgcheck=1
+Where:
 
-    gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-6
+- `<elevated-public-ip-address>` is the Elevated OFFICIAL public IP address (if you're not sure what this is, contact UKCloud Support)
 
-    [updates]
-
-    name=UKCloud CentOS Repository - Updates
-
-    baseurl=http://<elevated-public-ip-address>/centos/centos-updates
-
-    gpgcheck=1
-
-    gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-6
-
-    [extras]
-
-    name=UKCloud CentOS Repository - Extras
-
-    baseurl=http://<elevated-public-ip-address>/centos/centos-extras
-
-    gpgcheck=1
-
-    gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-6
-    ```
+- `<x>.<y>` is your version of CentOS
 
```

Comment:
For CentOS EPEL files, created a file called il3-epel.repo in /etc/yum.repos.d and populate it with the following

For CentOS6:

[epel]
name=Extra Packages for Enterprise Linux 6
baseurl=http://x.x.x.x/epel/
enabled=1
metadata_expire=7d
gpgcheck=1
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-6


For CentOS7:

[epel]
name=Extra Packages for Enterprise Linux 7
baseurl=http://x.x.x.x/epel-7/
enabled=1
metadata_expire=7d
gpgcheck=1
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-7